src/couch_replicator/src/couch_replicator_docs.erl

```diff
@@ -695,7 +695,8 @@ strip_credentials(Url) when is_binary(Url) ->
         "http\\1://\\2",
         [{return, binary}]);
 strip_credentials({Props}) ->
-    {lists:keydelete(<<"oauth">>, 1, Props)}.
+    Props0 = lists:keydelete(<<"oauth">>, 1, Props),
```

Comment:
Minor nit on variable names --  `Props1` might work a bit better here.

Usually when adding new code towards the middle or the bottom of the function, start adding 1,2,3 like so `Props, Props1=newoperation(Props)..,  ...use Props1 from now on...`. If there is a large function that uses `Props` in the body and we need to do modify it at the beginning of the function, then it's better to do ```function(Prop0 =...) -> Props = newoperation(Props0), ....```

Reply:
Thanks for your tips. Changed from `Props0` to `Props1`.